docs/guide/Titanium_SDK/Titanium_SDK_Guide/Appendices/tiapp.xml_and_timodule.xml_Reference.md

```diff
@@ -941,7 +941,7 @@ Because images are store in the Asset Catalog, reading image files manually from
 ::: danger ❗️ Experimental Feature
 This is an experimental feature with known issues:
 
-* [TIMOB-19806](https://jira.appcelerator.org/browse/TIMOB-19806)
+* [TIMOB-19806](https://jira-archive.titaniumsdk.com/TIMOB-19806)
```

Comment:
This ticket does not exist. I have no idea if it was private or deleted or what, but we don't have it.

So, I say remove the entire `:::` block.